compiler/rustc_passes/src/entry.rs

```diff
@@ -45,6 +45,8 @@ impl<'a, 'tcx> ItemLikeVisitor<'tcx> for EntryContext<'a, 'tcx> {
     fn visit_impl_item(&mut self, _impl_item: &'tcx ImplItem<'tcx>) {
         // Entry fn is never a trait item.
     }
+
+    fn visit_foreign_item(&mut self, _: &'tcx ForeignItem<'tcx>) {}
```

Comment:
```suggestion
    fn visit_foreign_item(&mut self, _: &'tcx ForeignItem<'tcx>) {
        // Entry fn is never a foreign item.
    }
```
comments are fun :shrug: